specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json

```diff
@@ -586,6 +594,12 @@
               "$ref": "#/definitions/DeploymentValidateResult"
             }
           },
+          "202": {
+            "description": "Accepted - The request has been accepted for processing and the operation will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/DeploymentValidateResult"
```

Comment:
A 202 response would not contain payload with it.

Reply:
Updating now.